tests/templates/kuttl/tls/check-tls.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+import trino
+import argparse
+import json
+import requests
+
+
+def get_http_connection(host, user):
+    return trino.dbapi.connect(
+        host=host,
+        port=8080,
+        user=user,
+        http_scheme='http',
+    )
+
+
+def get_https_connection(host, user, verify):
+    return trino.dbapi.connect(
+        host=host,
+        port=8443,
+        user=user,
+        http_scheme='https',
+        verify=verify
+    )
+
+
+def get_authenticated_https_connection(host, user, password, verify):
+    return trino.dbapi.connect(
+        host=host,
+        port=8443,
+        user=user,
+        http_scheme='https',
+        auth=trino.auth.BasicAuthentication(user, password),
+        verify=verify
+    )
+
+
+def test_query(conn, query):
+    cursor = conn.cursor()
+    cursor.execute(query)
+
+
+def test_query_failure(conn, query, error):
+    cursor = conn.cursor()
+    try:
+        cursor.execute(query)
```

Comment:
Need to raise an exception here if nothing fails

Reply:
done https://github.com/stackabletech/trino-operator/pull/244/commits/b507aeeb44dddd6e79016b8a4c423e576bdd7a31